frontend/src/pages/recommendations/RecommendationPage.tsx

```diff
@@ -16,26 +19,32 @@ import {
   recommendationsLanguagesFromNavigator,
 } from 'src/utils/recommendationsLanguages';
 import { useCurrentPoll } from 'src/hooks/useCurrentPoll';
+import { videoWithCriteriaFromRecommendation } from 'src/utils/entity';
 
-function VideoRecommendationPage() {
+function RecommendationsPage() {
   const { t } = useTranslation();
-  const prov: PaginatedVideoSerializerWithCriteriaList = {
+
+  const history = useHistory();
+  const location = useLocation();
+  const { criterias } = useCurrentPoll();
+  const [isLoading, setIsLoading] = useState(true);
+
+  const prov: PaginatedRecommendationList = {
     count: 0,
     results: [],
   };
-  const [videos, setVideos] = useState(prov);
-  const [isLoading, setIsLoading] = useState(true);
-  const location = useLocation();
-  const history = useHistory();
+
+  const [entities, setVideos] = useState(prov);
```

Comment:
Is it hard to rename `setVideos` into `setEntities` here?

Reply:
No, I just forgot to update the setter name. Good catch.